packages/terra-icon/scripts/src/README.md

```diff
@@ -1,10 +1,20 @@
 # Update all SVG icons
```

Comment:
A command could be added for each heading in this file with more than one command under it: `update-all-svg-icons`, `generate-meaningful-icons`, `generate-decorative-icons`

Reply:
Updated scripts here: 6e36b7964c05c7d5e6eea2f0ab72e6ded9ab4e9e

Reply:
Dove a little deeper into this:
- `npm run migrate-cerner-one-icons` won't work now because `npm generate-icon` isn't a script.
  - If `npm run generate-all-icons` is supposed to be part of `npm run migrate-cerner-one-icons`, wouldn't all steps in this document be completed in step 1?
- Under the "Step 2 - Generate meaningful icons heading:
  - `npm run generate-icon` isn't a thing now.
  - Once it's changed to reflect the new name for the command, everything but the last command is done directly by the command in step 1.
- `npm run replace-decorative-import` is part of the `generate-decorative-icons` command but isn't in the list of commands in this file to generate decorative icons

Reply:
Oh try pulling. I updated the readme and scripts.